python/tk_framework_desktopserver/shotgun/scripts/cache_commands.py

```diff
@@ -255,7 +255,7 @@ def cache(
     with open(arg_data_file, "rb") as fh:
         arg_data = cPickle.load(fh)
 
-    sys.path = arg_data["sys_path"]
+    sys.path.extend(arg_data["sys_path"])
```

Comment:
should we prepend everything instead, to make it takes precedence over anything that is in the path?